docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+
+## The MacBook's screen freezes, fans run up, and then it turns off
+
+This is a T2 panic; try a patched kernel (if you're not using one already), or boot macOS and check the crashlog. You may need a custom ISO. You can also try adding the ``efi=noruntime`` option to GRUB's command line paramaters.
+
+## The WiFi GUI is not working
+
+Try patched kernel `5.15.12` instead of later versions.
+
+## How do I get backlight working?
```

Comment:
also could be a matter of their gui not using it, checking in /sys/class/leds will show if the kernel side of it is working